src/NServiceBus.AzureStoragePersistence.AcceptanceTests/Infrastructure/EndpointTemplates/DefaultServer.cs

```diff
@@ -26,54 +27,61 @@ public DefaultServer(List<Type> typesToInclude)
             this.typesToInclude = typesToInclude;
         }
 
-        public BusConfiguration GetConfiguration(RunDescriptor runDescriptor, EndpointConfiguration endpointConfiguration, IConfigurationSource configSource, Action<BusConfiguration> configurationBuilderCustomization)
+        public Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointConfiguration, IConfigurationSource configSource, Action<EndpointConfiguration> configurationBuilderCustomization)
         {
-            var settings = runDescriptor.Settings;
-
-            LogManager.UseFactory(new ContextAppender(runDescriptor.ScenarioContext, endpointConfiguration.EndpointName));
-
             var types = GetTypesScopedByTestClass(endpointConfiguration);
 
             typesToInclude.AddRange(types);
 
-            var builder = new BusConfiguration();
+            var builder = new EndpointConfiguration();
 
             builder.EndpointName(endpointConfiguration.EndpointName);
-            builder.TypesToScan(typesToInclude);
+            builder.TypesToIncludeInScan(typesToInclude);
             builder.CustomConfigurationSource(configSource);
             builder.EnableInstallers();
-            builder.DefineTransport(settings, endpointConfiguration.BuilderType);
-            builder.DefineTransactions(settings);
-            builder.DefineBuilder(settings);
-            builder.RegisterComponents(r =>
-            {
-                r.RegisterSingleton(runDescriptor.ScenarioContext.GetType(), runDescriptor.ScenarioContext);
-                r.RegisterSingleton(typeof(ScenarioContext), runDescriptor.ScenarioContext);
-            });
 
-       
-            var serializer = settings.GetOrNull("Serializer");
+            builder.DisableFeature<TimeoutManager>();
+            builder.DisableFeature<SecondLevelRetries>();
+            builder.DisableFeature<FirstLevelRetries>();
 
-            if (serializer != null)
-            {
-                builder.UseSerialization(Type.GetType(serializer));
-            }
-            builder.DefinePersistence(settings);
+            builder.RegisterComponents(r => { RegisterInheritanceHierarchyOfContextOnContainer(runDescriptor, r); });
+
+            var connectionString = Environment.GetEnvironmentVariable("AzureStoragePersistence.ConnectionString");
+            builder.UsePersistence<AzureStoragePersistence>().ConnectionString(connectionString);
+
+            // Use this to disable synchronization storage by providing an empty implementation
+            // so that when the container tries to resolve an instance if ISynchronizedStorage it
+            // doesn't throw a ComponentNotRegisteredException exception
+            // This is hacky and we need a proper way around this.
+            builder.EnableFeature<FakeSynchronizationStorage>();
```

Comment:
We can use the newly merged fix for this. I'll make sure a new release is pushed to Myget and delete this code.